diff --git a/node_modules/next-contentlayer/dist/hooks/useLiveReload.js b/node_modules/next-contentlayer/dist/hooks/useLiveReload.js
index a5dfff7..970f1ee 100644
--- a/node_modules/next-contentlayer/dist/hooks/useLiveReload.js
+++ b/node_modules/next-contentlayer/dist/hooks/useLiveReload.js
@@ -1,5 +1,5 @@
-import { addMessageListener } from 'next/dist/client/dev/error-overlay/websocket.js';
-import { useRouter } from 'next/router.js';
+// import { addMessageListener } from 'next/dist/client/dev/error-overlay/websocket.js';
+// import { useRouter } from 'next/router.js';
 // NOTE use ESM/CommonJS compat import here until resolved: https://github.com/facebook/react/issues/11503
 import React from 'react';
 /**
