diff --git a/node_modules/next-seo/lib/types.d.ts b/node_modules/next-seo/lib/types.d.ts
index dd2144b..d3f5158 100644
--- a/node_modules/next-seo/lib/types.d.ts
+++ b/node_modules/next-seo/lib/types.d.ts
@@ -365,6 +365,7 @@ export interface NextSeoProps {
     additionalMetaTags?: ReadonlyArray<MetaTag>;
     additionalLinkTags?: ReadonlyArray<LinkTag>;
     children?: never;
+    logo?: any;
 }
 export interface DefaultSeoProps {
     dangerouslySetAllPagesToNoIndex?: boolean;
